simpleflow/command.py

```diff
@@ -104,14 +104,14 @@ def get_workflow_type(domain_name, workflow_class):
 def load_input(input_fp):
     if input_fp is None:
         input_fp = sys.stdin
-    input = json.load(input_fp)
+    input = json_loads_or_raw(input_fp)
```

Comment:
Oops, those changes with collide with #265 where I moved everything to the `swf.format` module (`swf.format.decode(input_fp)` in that case).